config/routes.rb

```diff
@@ -38,7 +38,7 @@
     get "/debug" => "debug#index", as: :debug_document
 
     get "/contacts" => "contacts#index"
-    post "/contacts" => "contacts#index"
+    post "/insert-contact" => "contacts#insert"
```

Comment:
This feels like it's be more consistent as `contacts/insert`

I'm wondering whether we intend to use ContactsController for the no-js experience. If not I think we could call this `contacts-embed` to be consistent with `video-embed`. I'm expecting they might be separate controllers as I don't think you'd need the document specific information.

Reply:
`contacts-embed` sounds good to me. I think `contacts/insert` would be somewhat unprecedented in our routes file. The controller does currently handle the no-JS experience, but I'm not sure how this will scale to the new design.